Resources/config/client.xml

```diff
@@ -12,6 +12,7 @@
             <argument>%ftrrtf_rollbar.notifier.client.options%</argument>
             <argument>%ftrrtf_rollbar.environment.options%</argument>
             <argument type="service" id="ftrrtf_rollbar.helper.user"/>
+            <argument type="expression">service(parameter('ftrrtf_rollbar.notifier.client.check_ignore_function_provider'))</argument>
```

Comment:
Unfortunately, expression works only from symfony 2.4, but bundle still has dependency with symfony `>=2.3` because it's long term support release http://symfony.com/roadmap?version=2.3#checker